apps/qolsysgw/mqtt/updater.py

```diff
@@ -444,6 +445,7 @@ class MqttWrapperQolsysSensor(MqttWrapper):
         QolsysSensorKeypad: 'safety',
         QolsysSensorMotion: 'motion',
         QolsysSensorSiren: 'safety',
+        QolsysSensorShock: 'vibration',
```

Comment:
```suggestion
        QolsysSensorShock: 'vibration',
        QolsysSensorSiren: 'safety',
```

for alphabetic order